### PR TITLE
Adicionado a aventura do assaltante

### PR DESCRIPTION
**Pull request by luizfelippe**

### CODE REVIEW

**Comment by luizfelippe** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Unfortunately, pull requests adding more than 100 lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to add 100 lines or fewer.

**Comment by luizfelippe** [comment]
Please review this, @casey-collab

File: portugues/ladrao/assalto.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+O ladrão chega com a arma apontando no seu peito e fala "vou te roubei". O que você faz?
+
+[Dou uma esquivada estilo Bruce Lee e retruco "vou te peguei"](ladrao/res/bruce.md)
```
Comment (casey-collab) [inline portugues/ladrao/assalto.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: portugues/ladrao/assalto.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+[Peço licença, tiro a ropa, me cubro de margarina e farinha e me jogo na frigideira](ladrao/res/panqueca.md)
```
Comment (casey-collab) [inline portugues/ladrao/assalto.md]: This link is also invalid.

File: portugues/ladrao/assalto.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+[Ele já te roubou](ladrao/res/gg.md)
```
Comment (casey-collab) [inline portugues/ladrao/assalto.md]: This link is also invalid.

File: portugues/ladrao/res/bruce.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+                            __--~~--_     _--~~--__
```
Comment (casey-collab) [inline portugues/ladrao/res/bruce.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: portugues/ladrao/res/gg.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+Ele já te roubou fera, não tem mais o que fazer.
```
Comment (casey-collab) [inline portugues/ladrao/res/gg.md]: This file has also been added to a directory that already contained a page.

File: portugues/ladrao/res/panqueca.md [hunk 6]
```diff
@@ -0,0 +1 @@
+Wat?
```
Comment (casey-collab) [inline portugues/ladrao/res/panqueca.md]: This file has also been added to a directory that already contained a page.

File: portugues/ladrao/res/bruce.md (hunk 4 above)
Comment (casey-collab) [inline portugues/ladrao/res/bruce.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: portugues/ladrao/res/panqueca.md (hunk 6 above)
Comment (casey-collab) [inline portugues/ladrao/res/panqueca.md]: This page is also unreachable.

File: portugues/ladrao/res/gg.md (hunk 5 above)
Comment (casey-collab) [inline portugues/ladrao/res/gg.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by luizfelippe** [comment]
Please review this, @casey-collab

File: portugues/ladrao/assalto.md [hunk 7]
```diff
@@ -0,0 +1,7 @@
+O ladrão chega com a arma apontando no seu peito e fala "vou te roubei". O que você faz?
+
+[Dou uma esquivada estilo Bruce Lee e retruco "vou te peguei"](res/bruce.md)
```
Comment (casey-collab) [inline portugues/ladrao/assalto.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: portugues/ladrao/assalto.md [hunk 8]
```diff
@@ -0,0 +1,7 @@
...
+
+[Peço licença, tiro a ropa, me cubro de margarina e farinha e me jogo na frigideira](res/panqueca.md)
```
Comment (casey-collab) [inline portugues/ladrao/assalto.md]: This link is also invalid.

File: portugues/ladrao/assalto.md [hunk 9]
```diff
@@ -0,0 +1,7 @@
...
+
+[Ele já te roubou](res/gg.md)
```
Comment (casey-collab) [inline portugues/ladrao/assalto.md]: This link is also invalid.

File: portugues/ladrao/gg/gg.md [hunk 10]
```diff
@@ -0,0 +1,5 @@
+Ele já te roubou fera, não tem mais o que fazer.
```
Comment (casey-collab) [inline portugues/ladrao/gg/gg.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: portugues/ladrao/panqueca/panqueca.md [hunk 11]
```diff
@@ -0,0 +1 @@
+Wat?
```
Comment (casey-collab) [inline portugues/ladrao/panqueca/panqueca.md]: This page is also unreachable.

File: portugues/ladrao/bruce/bruce.md [hunk 12]
```diff
@@ -0,0 +1,34 @@
+                            __--~~--_     _--~~--__
```
Comment (casey-collab) [inline portugues/ladrao/bruce/bruce.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by luizfelippe** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :icecream:

**Pull request merged**